lib/check_archived_flags.rb

```diff
@@ -0,0 +1,5 @@
+module CheckArchivedFlags
+  NONE = 0
+  TRASHED = 1
+  UNCONFIRMED = 2
+end
```

Comment:
Sawy, the best way to implement that is as a standalone module (like https://github.com/meedan/check-api/blob/develop/lib/error_codes.rb), not a module to be included in all classes.

Reply:
Done